realm/realm-library/src/androidTest/java/io/realm/OrderedCollectionChangeSetTests.java

```diff
@@ -147,9 +160,38 @@ private void modifyObjects(Realm realm, int... indices) {
     }
 
     private void moveObjects(Realm realm, int originAge, int newAge) {
-        realm.where(Dog.class).equalTo(Dog.FIELD_AGE, originAge).findFirst().setAge(newAge);
         if (type == ObservablesType.REALM_LIST) {
-            reorderRealmList(realm);
+            RealmList<Dog> dogs = realm.where(Owner.class).findFirst().getDogs();
```

Comment:
Please add a comment about what this method does

Reply:
simplified the code and added some comments.